Neos.Flow/Configuration/Objects.yaml

```diff
@@ -257,6 +257,12 @@ Neos\Flow\Http\Component\ComponentChain:
     1:
       setting: Neos.Flow.http.chain
 
+Neos\Flow\Http\Middleware\MiddlewaresChain:
+  factoryObjectName: Neos\Flow\Http\Middleware\MiddlewaresChainFactory
+  arguments:
+    1:
+      setting: Neos.Flow.http.middlewares
```

Comment:
Maybe we can even use the old component chain config directly, if we handle the order of components correctly